test/cppsuite/CMakeLists.txt

```diff
@@ -54,6 +54,18 @@ create_test_executable(run
     CXX
 )
 
+create_test_executable(csuite_style_example_test
+    SOURCES test_harness/connection_manager.cxx
+            test_harness/thread_manager.cxx
+            test_harness/util/logger.cxx
+            test_harness/util/scoped_types.cxx
+            test_harness/workload/random_generator.cxx
+            tests/csuite_style_example_test.cxx
+    FLAGS -std=c++11
+    INCLUDES ${CMAKE_CURRENT_SOURCE_DIR}
+    CXX
+)
+
```

Comment:
Also did you want the `csuite_style_example_test` to run during our regression tests? If so you will also need to need to add the following to run:
```
add_test(NAME csuite_style_example COMMAND csuite_style_example_test)
set_tests_properties(csuite_style_example PROPERTIES LABELS "check")
```

Reply:
Yes, this would be great, added in 9e510ae477e961ae2078139329c1949ebc0dc5c3.